### PR TITLE
Document enableNdkScopeSync option for RN

### PR DESCRIPTION
**Pull request by marandaneto**
similar to Android https://docs.sentry.io/platforms/android/enriching-events/scopes/#configuring-the-scope

### CODE REVIEW

File: src/includes/enriching-events/scopes/configure-scope/react-native.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  scope.setUser({
+    id: 42,
+    email: "john.doe@example.com",
+  });
+});
+```
+
+### Scope Synchronization
+
+If you want to set context data in the JS layer, then receive that context data when a native (C/C++) crash happens, you need to synchronize the JS `Scope` with the native `Scope`. This feature was introduced in version `2.0.0` of the React-Native SDK and requires an opt-in:
```
Comment (imatwawana) [inline src/includes/enriching-events/scopes/configure-scope/react-native.mdx:13]:
```suggestion
If you want to set context data in the JS layer and then receive that context data when a native (C/C++) crash happens, you need to synchronize the JS `Scope` with the native `Scope`. This feature was introduced in version `2.0.0` of the React-Native SDK and requires an opt-in:
```

File: src/platforms/react-native/troubleshooting.mdx [hunk 2]
```diff
@@ -106,3 +106,7 @@ module.exports = {
 As this applies to **all** class and function names — not just React components — this could increase your JS bundle size.
 
 </Note>
+
+## Missing context on Android NDK events
```
Comment (imatwawana) [inline src/platforms/react-native/troubleshooting.mdx:110]:
```suggestion
## Missing Context on Android NDK Events
```

File: src/platforms/react-native/troubleshooting.mdx [hunk 3]
```diff
@@ -106,3 +106,7 @@ module.exports = {
...
+
+In case it is missing context on Android NDK events as breadcrumbs, user, etc..., You've to enable the [Scope Synchronization](/platforms/react-native/enriching-events/scopes/#scope-synchronization) feature.
```
Comment (imatwawana) [inline src/platforms/react-native/troubleshooting.mdx:112]:
```suggestion
If the context is missing on Android NDK events such as breadcrumbs, user, and so on, you need to enable the [Scope Synchronization](/platforms/react-native/enriching-events/scopes/#scope-synchronization) feature.
```

**Review by imatwawana — APPROVED** [review]
A few style and wording edits

**Comment by marandaneto** [comment]
thanks @imatwawana

**Pull request merged**